CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -280,14 +280,8 @@ public function upgrade_4_7_14($rev) {
    * @param string $rev
    */
   public function upgrade_4_7_15($rev) {
-    $this->addTask('CRM-19723 - Add icon column to civicrm_option_value', 'addColumn',
-      'civicrm_option_value', 'icon', "varchar(255) COMMENT 'crm-i icon class' DEFAULT NULL");
     $this->addTask('CRM-19626 - Add min_amount column to civicrm_price_set', 'addColumn',
```

Comment:
For posterity: We discussed this one on Mattermost. This step is in an odd position where the `4.7.15` uses a SQL-backed upgrade and `4.7.16` uses a newer PHP-backed upgrade. However, testing shows that they end up in the same place, and they still appear mutually compatible, and the `4.7.16` code is better.